src/net.cpp

```diff
@@ -1659,13 +1537,14 @@ void CConnman::SocketHandlerConnected(const std::vector<CNode*>& nodes,
     }
 }
 
-void CConnman::SocketHandlerListening(const std::set<SOCKET>& recv_set)
+void CConnman::SocketHandlerListening(Sock::WaitData& what)
```

Comment:
af45eee `what` is an in-param
```suggestion
void CConnman::SocketHandlerListening(const Sock::WaitData& what);
```

Reply:
Done.